mkdocs.yml

```diff
@@ -109,8 +109,9 @@ nav:
                   - reference/configuration/Models/CsvScriptModel/index.md
               - LocalMergeModel:
                   - reference/configuration/Models/LocalMergeModel/index.md
-          - Table:
```

Comment:
@jpjsonic Do you know why this would happen?

Reply:
Hmm, I'm not sure. That's pretty surprising behavior. I probably need to refactor the mkdocs parser of our schema in general it's a little confusing to make make changes to and test right now.

Reply:
I'll spend some time with it and see why it's doing that.

Reply:
Per our thread in slack- it seems like this is pertaining to having two pydantic models named Table. We can get rid of the plotly trace prop for table since we dont want to support it anyways.